sdk/spring/spring-cloud-azure-feature-management/src/test/java/com/azure/spring/cloud/feature/management/filters/recurrence/RecurrenceEvaluatorTest.java

```diff
@@ -0,0 +1,534 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.feature.management.filters.recurrence;
+
+import com.azure.spring.cloud.feature.management.implementation.timewindow.TimeWindowFilterSettings;
+import com.azure.spring.cloud.feature.management.implementation.timewindow.recurrence.RecurrenceEvaluator;
+import com.azure.spring.cloud.feature.management.implementation.models.Recurrence;
+import com.azure.spring.cloud.feature.management.implementation.models.RecurrencePattern;
+import com.azure.spring.cloud.feature.management.implementation.models.RecurrenceRange;
+import org.junit.jupiter.api.Test;
+
+import java.time.ZonedDateTime;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RecurrenceEvaluatorTest {
+
+    @Test
+    public void dailyTrue() {
+        final ZonedDateTime now1 = ZonedDateTime.parse("2023-09-02T00:00:00+08:00");
+        final TimeWindowFilterSettings settings1 = new TimeWindowFilterSettings();
+        final RecurrencePattern pattern1 = new RecurrencePattern();
+        final RecurrenceRange range1 = new RecurrenceRange();
+        final Recurrence recurrence1 = new Recurrence();
+        pattern1.setType("Daily");
```

Comment:
May I know the reason why sometimes there is index in the variabe name and sometimes there isn't. e.g. in the `dailyEndDateFalse` test. I prefer to remove all the index, `settings` is fine.

Reply:
Yes. This is because I used to put a few test cases in one function. For example, these test cases is all related to weekly, then the other is related to relativeMonthly in very earlier implementation. So I will have some variables named settings1, setting2,... 
Then updated to split these cases to multi functions. forgot to update the variables name. 
Thannks for catching this. Updated.